lib/Yam/Agama/Pom/GrubMenuBasePage.pm

```diff
@@ -24,7 +24,20 @@ sub edit_current_entry {
 }
 
 sub select_first_entry {
-    send_key("ret");
+    if (get_var('WORKAROUND_BSC_1230121')) {
```

Comment:
we don't need conditional code in the page object, it would be the same mess that put it in the test itself, only in the distri pattern makes sense.
You can see this example: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20394/files
It doesn't use that big list of openQA api command, but methods covering those actions.

Reply:
do you mean to create a new eg `lib/Yam/Agama/Pom/GrubMenuBasePageBug1230121.pm` and use it inside `lib/Distribution/Sle/AgamaDevel.pm` in `sub get_grub_menu_installed_system`?

Reply:
yes, around there (I don't know exactly) but probably it should use the same way with `get_grub_menu_base` implementing a different base only for sle.

Reply:
Moved condition under `Distribution::Sle::AgamaDevel` and created `lib/Yam/Agama/Pom/GrubMenuBasePageBug1230121.pm`